packages/dashboard/src/components/Section/Section.css

```diff
@@ -0,0 +1,3 @@
+.react-grid-item {
+  box-shadow: none;
+}
```

Comment:
Can this go in Section.styles instead?

Reply:
fixed.

Reply:
Fixed.

---
In reply to: [210772001](https://github.com/OfficeDev/office-ui-fabric-react/pull/5899#discussion_r210772001) [](ancestors = 210772001)